### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by fisker**

### CODE REVIEW

**Comment by fisker** [comment]
@thorn0 @sosukesuzuki Any of you know how to remove "Travis CI"? The file has been deleted

File: src/attach.ts [hunk 1]
```diff
@@ -108,7 +108,7 @@ function initNodeTable(nodeTable: NodeTable, node: YamlUnistNode): void {
   }
 
   if ("children" in node) {
-    (node.children as Array<(typeof node.children)[number]>).forEach(child => {
+    (node.children as Array<typeof node.children[number]>).forEach(child => {
```
Comment (fisker) [inline src/attach.ts:111]: @thorn0 @sosukesuzuki Is this expected? As one not familiar with TS, it's unreadable to me. I can't find when we changed this.

Comment (thorn0) [reply src/attach.ts:111]: Noticed this too. Looks bad.

Comment (thorn0) [reply src/attach.ts:111]: Pretty sure it was me in https://github.com/prettier/prettier/pull/6605, not intentionally.

Comment (fisker) [reply src/attach.ts:111]: Strange thing is nobody complain about it for 3 years. I even thought this changes syntax.

**Pull request merged**